 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 17e369e..b55f8d4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,7 +51,7 @@ if(MSVC)
     add_compile_options(/W4 /WX /wd4251 /wd4100)
 else()
     # TODO -Wno-comment should be removed when ODAUTIL-169 is fixed
-    add_compile_options(-Wall -Wno-unknown-pragmas -Wno-reorder -Werror -Wextra -pedantic -Wno-comment)
+    # add_compile_options(-Wall -Wno-unknown-pragmas -Wno-reorder -Werror -Wextra -pedantic -Wno-comment)
 endif()
 
 # check for conan
@@ -86,6 +86,7 @@ IF (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     add_compile_options("$<$<CONFIG:DEBUG>:-Og>")
     message("Disable optimization")
 ENDIF(CMAKE_COMPILER_IS_GNUCC)
+add_compile_definitions(DEV_ESSENTIAL_DISABLE_MAKE_RESULT_CHECK_RESERVED_ZERO)
 
 include(scripts/cmake/enable_multicore_compilation.cmake)
 include(scripts/cmake/use_integrated_debug_symbols.cmake)